bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/HorizonsDesignation.scala

```diff
@@ -1,20 +1,34 @@
 package edu.gemini.spModel.core
 
+
 import scalaz._, Scalaz._
 
-/** 
+/**
  * Unique Horizons designation, which should allow for reproduceable ephemeris queries <b>if</b> the
  * values passed to the constructors are extracted correctly from search results. See `horizons.md`
  * in the bundle source for more information.
  */
 sealed abstract class HorizonsDesignation(val queryString: String)
   extends Product with Serializable {
     def des: String // designation, human readable
+
+    import edu.gemini.spModel.core.HorizonsDesignation._
+
+    /** Exports an HorizonsDesignation to a String in a format that can be read
+      * by the `HorizonsDesignation.read` method.
+      */
+    def show: String =
```

Comment:
The ephemeris robot will use `show` to come up with a string representation of the horizons id to use for the ephemeris filename.   Later when it runs again it will use `read` to convert the filename back to a horizons id so it can do housekeeping on the ephemeris files.